apps/wallet/package.json

```diff
@@ -89,6 +89,7 @@
         "buffer": "^6.0.3",
         "classnames": "^2.3.1",
         "eslint-plugin-react": "^7.31.8",
+        "events": "^3.3.0",
```

Comment:
We probably should use a different event module that isn't node compatible but will be faster / smaller. I've used `mitt` and `nanoevents` before.

Reply:
Changed it with `mitt`